roles/openshift_logging_mux/vars/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 __latest_mux_version: "3_5"
 __allowed_mux_versions: ["3_5", "3_6"]
+openshift_logging_mux_buffer_type: "file"
```

Comment:
prefix this with "__" please
we will also need to make sure we're loading and setting the `openshift_logging_mux_buffer_type` fact afterwards too